FWCore/PluginManager/src/PauseMaxMemoryPreloadSentry.cc

```diff
@@ -0,0 +1,11 @@
+#include "PauseMaxMemoryPreloadSentry.h"
+
+// By default do nothing, but add "hooks" that MaxMemoryPreload can
+// override with LD_PRELOAD
+void pauseMaxMemoryPreload() {}
+void unpauseMaxMemoryPreload() {}
```

Comment:
Maybe better to make these `extern "C"`

Reply:
Actually no, it shouldn't matter here, where we and our C++ code is responsible of both ends.